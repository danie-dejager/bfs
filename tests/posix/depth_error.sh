! bfs_diff inaccessible -depth
