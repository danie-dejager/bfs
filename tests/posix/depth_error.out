inaccessible
inaccessible/dir
inaccessible/file
inaccessible/link
